File: core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ambari/AmbariHostsWithNames.java
package com.sequenceiq.cloudbreak.service.cluster.ambari;

import com.sequenceiq.ambari.client.AmbariClient;
import com.sequenceiq.cloudbreak.domain.Stack;
import com.sequenceiq.cloudbreak.service.cluster.flow.AmbariClientPollerObject;

import java.util.List;

public class AmbariHostsWithNames extends AmbariClientPollerObject {

    private final List<String> hostNames;

    public AmbariHostsWithNames(Stack stack, AmbariClient ambariClient, List<String> hostNames) {
        super(stack, ambariClient);
        this.hostNames = hostNames;
    }

    public List<String> getHostNames() {
        return hostNames;
    }
}

File: core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/AmbariHostsWithNames.kt
package com.sequenceiq.cloudbreak.service.cluster.flow

import com.sequenceiq.ambari.client.AmbariClient
import com.sequenceiq.cloudbreak.domain.Stack

class AmbariHostsWithNames(stack: Stack, ambariClient: AmbariClient, val hostNames: List<String>) : AmbariClientPollerObject(stack, ambariClient)
